src/NServiceBus.Core/Pipeline/FastExpressionCompiler.g.cs

```diff
@@ -0,0 +1,3856 @@
+﻿/*
+The MIT License (MIT)
+
+Copyright (c) 2016 Maksim Volkau
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included AddOrUpdateServiceFactory
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+/*
+ v1.10.1 from https://github.com/dadhi/FastExpressionCompiler
```

Comment:
@danielmarbach latest version I see on that repo is 1.7.1. Where did you take that version number from?

Reply:
![image](https://user-images.githubusercontent.com/174258/44035253-5ee19dd4-9f0f-11e8-9533-0a7b05a270b3.png)

Reply:
@danielmarbach thanks, didn't see that. I was just looking at the releases and branches and didn't find this version there.